metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/status_params.py

```diff
@@ -80,10 +81,14 @@
 # REST
 metron_rest_port = config['configurations']['metron-rest-env']['metron_rest_port']
 
-# UI
-metron_management_ui_port = config['configurations']['metron-management-ui-env']['metron_management_ui_port']
+# Alerts UI
+metron_alerts_ui_host = default("/clusterHostInfo/metron_alerts_ui_hosts", [hostname])[0]
```

Comment:
I needed to somehow find the hosts running each service and I knew it was contained in this `clusterHostInfo` configuration.  But it was really difficult to uncover what values Ambari keeps in this `clusterHostInfo` configuration.  I have not been able to find any documentation on this.  

I actually had to add some debug statement to a live instance of Ambari to find out what values are stored here and how they are named.  Fun, fun.

For the record, here is what is stored in `clusterHostInfo` when spinning up the current state of Full Dev.
```
{  
   'snamenode_host':[  
      'node1'
   ],
   'metron_alerts_ui_hosts':[  
      'node1'
   ],
   'nm_hosts':[  
      'node1'
   ],
   'drpc_server_hosts':[  
      'node1'
   ],
   'ambari_server_use_ssl':[  
      'false'
   ],
   'all_ping_ports':[  
      '8670'
   ],
   'all_hosts':[  
      'node1'
   ],
   'rm_host':[  
      'node1'
   ],
   'kafka_broker_hosts':[  
      'node1'
   ],
   'slave_hosts':[  
      'node1'
   ],
   'metron_profiler_hosts':[  
      'node1'
   ],
   'storm_ui_server_hosts':[  
      'node1'
   ],
   'all_racks':[  
      '/default-rack'
   ],
   'all_ipv4_ips':[  
      '127.0.0.1'
   ],
   'app_timeline_server_hosts':[  
      'node1'
   ],
   'hs_host':[  
      'node1'
   ],
   'ambari_server_port':[  
      '8080'
   ],
   'metron_rest_hosts':[  
      'node1'
   ],
   'metron_management_ui_hosts':[  
      'node1'
   ],
   'es_master_hosts':[  
      'node1'
   ],
   'metron_parsers_hosts':[  
      'node1'
   ],
   'kibana_master_hosts':[  
      'node1'
   ],
   'metron_enrichment_master_hosts':[  
      'node1'
   ],
   'hbase_rs_hosts':[  
      'node1'
   ],
   'namenode_host':[  
      'node1'
   ],
   'nimbus_hosts':[  
      'node1'
   ],
   'hbase_master_hosts':[  
      'node1'
   ],
   'metron_indexing_hosts':[  
      'node1'
   ],
   'ambari_server_host':[  
      'node1'
   ],
   'zookeeper_hosts':[  
      'node1'
   ],
   'supervisor_hosts':[  
      'node1'
   ]
}
```